src/main/java/com/xiao/users/config/AppConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.xiao.users.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+public class AppConfig {
```

Comment:
consider stucture of kafka config

Reply:
Renamed AppConfig to KafkaConfigProperties